source/common/runtime/runtime_impl.cc

```diff
@@ -286,7 +286,7 @@ double SnapshotImpl::getDouble(const std::string& key, double default_value) con
   }
 }
 
-bool SnapshotImpl::getBoolean(absl::string_view key, bool& value) const {
```

Comment:
This change doesn't make sense to me given the function body?

Reply:
Yikes, I was asleep at the wheel for that one.. sorry.

Had to do a bit more surgery on the snapshot functions to get that working as expected- should be in a better place now.